java/code/src/com/redhat/rhn/common/db/datasource/SelectMode.java

```diff
@@ -79,6 +79,15 @@ public void addElaborator(CachedStatement q) {
         elaborators.add(q);
     }
 
+    /**
+     * Remove an elaborator query.
+     * @param queryName Elaborator query name to remove.
+     */
+    public void removeElaboratorByName(String queryName) {
```

Comment:
I think instead of queryName it should be called "elaboratorName", since we want to remove the elaborator

Reply:
ok, I'm still working on it for trying a way to set the status "?" instead of green